pkgs/tools/security/sss-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sss-cli";
+  version = "0.1.1";
```

Comment:
Looks like the version number doesn't match the one in the commit message/pr title